instant-markdown-d

```diff
@@ -59,7 +59,9 @@ function httpHandler(req, res) {
         newHTML += data;
       });
       gfm.on('exit',function(ecode){
-        socket.emit('newContent', newHTML);
+        if (socket) {
+          socket.emit('newContent', newHTML);
+        }
```

Comment:
Just curious why this was necessary? Did you encounter cases where it wasn't defined?